src/settings.py

```
@@ -0,0 +1,10 @@
+DEBUG = False
```

中: サンプルの settings をコピーするのであれば、サンプル側に上の記述をいれてこのモジュールは削除したほうが二重管理しなくてよいのでは?継承しなくてよいのならしないほうが単純になるかなと。あと感覚的な話ですが、 `except: pass` するような記述の設定ファイルは不要な気が..

---

あんまりよく考えていなかったです。sample用意するならそうですね。 こちらのモジュールは削除します。